app/products/calls/components/channel_info_start/channel_info_start_button.tsx

```diff
@@ -84,12 +84,27 @@ const ChannelInfoStartButton = ({
             setConnecting(false);
             dismissChannelInfo();
         }
-    }, [isLimitRestricted, alreadyInCall, dismissChannelInfo, intl, serverUrl, channelId, isACallInCurrentChannel, otherParticipants]);
+    }, [
+        alreadyInCall,
+        isLimitRestricted,
+        intl,
+        otherParticipants,
+        isAdmin,
+        isHost,
+        serverUrl,
+        channelId,
+        dismissChannelInfo,
+        limitRestrictedInfo,
+        isACallInCurrentChannel,
+        joining,
+        starting,
+    ]);
 
     const [tryJoin, msgPostfix] = useTryCallsFunction(toggleJoinLeave);
+    const onPress = usePreventDoubleTap(tryJoin);
 
     const joinText = intl.formatMessage({id: 'mobile.calls_join_call', defaultMessage: 'Join call'});
-    const startText = intl.formatMessage({id: 'mobile.calls_start_call', defaultMessage: 'Start call'});
+    const startText = intl.formatMessage({id: 'mobile.calls_start_call', defaultMessage: 'Start Call'});
```

Comment:
Intended?

Reply:
Same answer.

@cwarnermm it would be great to know also our policy on casing for these strings, to either keep this as is, or update the `en.json` file.

Reply:
I recommend **Sentence case** for strings that aren't button labels. @matthewbirtch - Do you agree?